dev-requirements.txt

```diff
@@ -33,7 +33,7 @@ git+https://github.com/NVIDIA/apex.git@master
 # YAML manipulation
 ruamel.yaml
 
-mathy_pydoc>=0.6.7,<0.8.0
+git+https://github.com/NiklasRosenstein/pydoc-markdown.git@master
```

Comment:
Should we pin to a particular commit so we have a deterministic dependency?

Reply:
Good idea